### PR TITLE
Integrate ga segment to collect telemetry data

### PR DESCRIPTION
**Pull request by VkoHov**

### CODE REVIEW

**Review by roubkar — APPROVED** [review]

File: aim/web/ui_v2/src/services/models/metrics/metricsAppModel.ts [hunk 1]
```diff
@@ -1988,13 +2093,19 @@ function onColumnsVisibilityChange(hiddenColumns: string[]) {
...
+    analytics.trackEvent('[MetricsExplorer][Table] Show all table columns');
+  }
 }
 
 function onTableDiffShow() {
   const sameValueColumns = model.getState()?.sameValueColumns;
   if (sameValueColumns) {
     onColumnsVisibilityChange(sameValueColumns);
   }
+  analytics.trackEvent('[ParamsExplorer][Table] Show table columns diff');
```
Comment (gorarakelyan) [inline aim/web/ui_v2/src/services/models/metrics/metricsAppModel.ts:2108]: Typo? MetricsExplorer

File: aim/web/ui_v2/src/services/models/metrics/metricsAppModel.ts [hunk 2]
```diff
@@ -2015,6 +2126,13 @@ function onColumnsOrderChange(columnsOrder: any) {
     setItem('metricsTable', encode(table));
     updateModelData(config);
   }
+  if (
+    isEmpty(columnsOrder?.left) &&
+    isEmpty(columnsOrder?.middle) &&
+    isEmpty(columnsOrder?.right)
+  ) {
+    analytics.trackEvent('[ParamsExplorer][Table] Reset table columns order');
```
Comment (gorarakelyan) [inline aim/web/ui_v2/src/services/models/metrics/metricsAppModel.ts:2134]: MetricsExplorer?

File: aim/web/ui_v2/src/services/models/params/paramsAppModel.ts [hunk 3]
```diff
@@ -723,6 +723,11 @@ function onColorIndicatorChange(): void {
     chart.isVisibleColorIndicator = !configData.chart.isVisibleColorIndicator;
     updateModelData({ ...configData, chart });
   }
+  analytics.trackEvent(
+    `[ParamsExplorer][Chart] ${
+      configData.chart.isVisibleColorIndicator ? 'Hide' : 'Show'
```
Comment (gorarakelyan) [inline aim/web/ui_v2/src/services/models/params/paramsAppModel.ts:728]: Enable/disable color indicator

File: aim/web/ui_v2/src/services/models/params/paramsAppModel.ts [hunk 4]
```diff
@@ -1137,6 +1166,7 @@ async function onBookmarkCreate({ name, description }: IBookmarkFormState) {
         });
     }
   }
+  analytics.trackEvent('[ParamsExplorer] Create bookmark state');
```
Comment (gorarakelyan) [inline aim/web/ui_v2/src/services/models/params/paramsAppModel.ts:1169]: Just "Create bookmark"?

File: aim/web/ui_v2/src/services/models/params/paramsAppModel.ts [hunk 5]
```diff
@@ -1256,6 +1288,7 @@ function onExportTableData(e: React.ChangeEvent<any>): void {
     type: 'text/csv;charset=utf-8;',
   });
   saveAs(blob, `params-${moment().format('HH:mm:ss · D MMM, YY')}.csv`);
+  analytics.trackEvent('[ParamsExplorer] Export to CSV');
```
Comment (gorarakelyan) [inline aim/web/ui_v2/src/services/models/params/paramsAppModel.ts:1291]: Export runs data to CSV

File: aim/web/ui_v2/src/services/models/runs/runsAppModel.ts [hunk 6]
```diff
@@ -443,6 +445,7 @@ function onExportTableData(e: React.ChangeEvent<any>): void {
     type: 'text/csv;charset=utf-8;',
   });
   saveAs(blob, `runs-${moment().format('HH:mm:ss · D MMM, YY')}.csv`);
+  analytics.trackEvent('[RunsExplorer][Table] Export to CSV');
```
Comment (gorarakelyan) [inline aim/web/ui_v2/src/services/models/runs/runsAppModel.ts:448]: Export runs data to CSV

File: aim/web/ui_v2/src/services/models/runs/runsAppModel.ts [hunk 7]
```diff
@@ -863,6 +866,13 @@ function onColumnsOrderChange(columnsOrder: any) {
     setItem('runsTable', encode(table));
     updateModelData(config);
   }
+  if (
+    isEmpty(columnsOrder?.left) &&
+    isEmpty(columnsOrder?.middle) &&
+    isEmpty(columnsOrder?.right)
+  ) {
+    analytics.trackEvent('[ParamsExplorer][Table] Reset table columns order');
```
Comment (gorarakelyan) [inline aim/web/ui_v2/src/services/models/runs/runsAppModel.ts:874]: Typo? RunsExplorer?

**Review by gorarakelyan — APPROVED** [review]
lgtm

**Pull request merged**